src/documentation/dsls/sql/statements/select.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: section
+title: "Beam SQL: SELECT Statement"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/statements/select/
+---
+
+# SELECT
```

Comment:
This should have sections for things like how GROUP BY is likely to result in grouping so that is where windowing and triggering are applied.